packages/config/src/test/stubs/layer2aWithDups.ts

```diff
@@ -343,17 +361,19 @@ export const layer2aWithDups: Layer2 = {
           address: EthereumAddress(
             '0xCd9D4988C0AE61887B075bA77f08cbFAd2b65068',
           ),
+          chain: 'ethereum',
           isVerified: true,
         },
         {
           name: 'L1DaiGateway',
           address: EthereumAddress(
             '0x10E6593CDda8c58a1d0f14C5164B376352a55f2F',
           ),
+          chain: 'ethereum',
           isVerified: true,
         },
       ],
     },
     risks: [],
   },
-}
+} as unknown as Layer2
```

Comment:
why?

Reply:
This object is used only for testing if we are able to extract different chains from permissions/chains. I don't want to change this big object every time I do a small change with data that is never touched